tket/src/Circuit/Boxes.cpp

```diff
@@ -489,8 +489,7 @@ bool ProjectorAssertionBox::is_equal(const Op &op_other) const {
   return m_.isApprox(other.m_);
 }
 
-StabiliserAssertionBox::StabiliserAssertionBox(
-    const PauliStabiliserList &paulis)
+StabiliserAssertionBox::StabiliserAssertionBox(const PauliStabiliserVec &paulis)
     : Box(OpType::StabiliserAssertionBox),
       paulis_(paulis),
```

Comment:
We need to check the coefficients are real.

Reply:
This is now handled by the call to `PauliTensor::is_real_negative()` in `add_assertion_operator()` (called as part of `generate_circuit()`).